index.html

```diff
@@ -13,11 +13,9 @@
 </head>
 <body>
   <main class="container">
-    <section class="introduction">
+    <div class="introduction">
       <header class="main-header">
-        <!-- flex-container -->
         <div class="toolbar"> 
```

Comment:
- [ ] As the previous reviewer mentioned, please know that you're building a real website, this means this bar on the very top isn't necessary it's in the figma design just to show you how it would look in a real smartphone, this is why you should remove it. 😁